actors/ppo_atari_pz.py

```diff
@@ -0,0 +1,1085 @@
+# Copyright 2022 AI Redefined Inc. <dev+cogment@ai-r.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import math
+from abc import ABC, abstractmethod
+from ast import literal_eval
+from typing import List, Tuple
+
+import cogment
+import numpy as np
+import torch
+from gym.spaces import Discrete, utils
+from torch.distributions.distribution import Distribution
+
+from cogment_verse import HumanDataBuffer, Model
+from cogment_verse.run.run_session import RunSession
+from cogment_verse.run.sample_producer_worker import SampleProducerSession
+from cogment_verse.specs import (
+    HUMAN_ACTOR_IMPL,
+    PLAYER_ACTOR_CLASS,
+    WEB_ACTOR_NAME,
+    AgentConfig,
+    EnvironmentConfig,
+    EnvironmentSpecs,
+    cog_settings,
+)
+
+torch.multiprocessing.set_sharing_strategy("file_system")
+
+log = logging.getLogger(__name__)
+
+# pylint: disable=E1102
+# pylint: disable=W0212
+# pylint: disable=W0223
+# pylint: disable=W0613
+# pylint: disable=W0612
+# pylint: disable=C0302
```

Comment:
please use the error name not the code, it facilitates readability

Reply:
Done